src/test/java/cucumber/setUp/CucumberContext.java

```diff
@@ -0,0 +1,64 @@
+package cucumber.setUp;
+
+import manager.PageFactoryManager;
+import org.apache.log4j.PropertyConfigurator;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.Listeners;
+import pages.HomePage;
+import reporting.TestListener;
+import utils.PropertiesHelper;
+import webDriverFactory.LocalWebDriverCreator;
+import webDriverFactory.RemoteWebDriverCreator;
+import webDriverFactory.WebDriverCreator;
+
+import static java.util.Optional.ofNullable;
+import static utils.PropertiesHelper.getInstance;
+
+@Listeners({TestListener.class})
+public class CucumberContext {
+  public static PropertiesHelper propertiesHelper;
+  public final int FIRST = 1;
+  public PageFactoryManager pageFactoryManager;
```

Comment:
You use this variable only once in the setUp method. It's better to use 
`homePage = new PageFactoryManager(driver).getPage(HomePage.class);`